Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -559,30 +569,38 @@ class KalmanFitter {
         materialInteractor(surface, state, stepper, postUpdate);
         // We count the processed state
         ++result.processedStates;
-      } else if (surface->surfaceMaterial() != nullptr) {
+        // Increase the number of holes count only when encoutering a
+        // measurement
+        result.measurementHoles = result.missedActiveSurfaces.size();
```

Comment:
Maybe I missed something, but I don't see a place where the `result.missedActiveSurfaces` is filled...

Reply:
It is filled line 610 if a surface has no measurement and either an associated detector element or associated material (this line didn't change from what it used to be).

Reply:
Ha, my bad. I thought the `missedActiveSurfaces` is newly added, but actually it's already there.